lib/Data/Database.php

```diff
@@ -542,7 +548,20 @@ protected function _getExpiredPastes($batchsize)
     private static function _exec($sql, array $params)
     {
         $statement = self::$_db->prepare($sql);
-        $result    = $statement->execute($params);
+        if (self::$_type === 'oci') {
+            // It is not possible to execute in the normal way if strlen($param) >= 4000
```

Comment:
Is this comment relevant here? It's more about the data type, is it? (not about how the parameters are now iterated in the code below? :thinking: )

If so, what is the "normal way" actually?

Reply:
I'd suggest to just remove that comment.

Reply:
I'm not 100% sure, but I had understood that oci requires the use of bindParam, then later bindValue for this comments reason? Ideally we have a comment to explain why this condition is needed.

~~~

Actually, since you suggested above that you prefer the bindValue way of binding the parameters, I tried to use this as the only way and at least SQLite in the unit testing is fine with that, so I'll just make this the default and remove the comment and if-else.